client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -62,18 +64,20 @@ const SiteSettingsSecurity = ( { site, siteId, siteIsJetpack, translate } ) => {
 
 	return (
 		<Main className="settings-security__main site-settings">
+			<QueryRewindStatus siteId={ siteId } />
```

Comment:
Shouldn't you use this method?
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/activity-log/index.jsx#L598

Reply:
Won't `<QueryRewindStatus />` return true always? Even if the network comes back and says rewind is *not* active?

Reply:
Perhaps that might be better. I'll try it.

Reply:
Fixed :-) Thanks.